README.md

```diff
@@ -42,6 +42,25 @@ Crystalball.foresee do |predictor|
 end
 ```
 
+## Map Generator
+
+There are different map generator strategies that can (and should) be used together for better predictions. Each one has its own benefits and drawbacks, so they should be configured to best fit your needs.
+
+- **CoverageStrategy** uses coverage information to detect which files are covered by the given spec (i.e. the files that, if changed, may potentially break the spec);
+- **AllocatedObjectsStrategy** looks for the files in which the objects created during the spec execution are defined. It is considerably slower than `CoverageStrategy`.
+
+## Predictor
+
+The predictor can also be customized with different strategies:
+
+- **AssociatedSpecs** needs to be configured with rules for detecting which specs should be on the prediction.
+  `predictor.use Crystalball::Predictor::AssociatedSpecs.new(from: %r{models/(.*).rb}, to: "./spec/models/%s_spec.rb")`
```

Comment:
Change it as per https://github.com/toptal/crystalball/pull/50

Reply:
and following https://github.com/toptal/crystalball/pull/51